### PR TITLE
Update to stable 23.10 backend version

### PR DESCRIPTION
**Pull request by dae**
Requires https://github.com/ankidroid/Anki-Android-Backend/pull/314

### CODE REVIEW

**Review by mikehardy — APPROVED** [review]
✅ ~~backend PR merged, waiting for CI to go quiet then will start release process there~~
✅ ~~backend PR release process started @ 1605UTC~~
✅ ~~Once artifacts are available on maven repo (likely 1905 or so UTC) can re-run CI here
maven repo availability check: https://repo1.maven.org/maven2/io/github/david-allison-1/anki-android-backend/~~

**Comment by david-allison** [comment]
on it

File: AnkiDroid/src/main/java/com/ichi2/libanki/Notetypes.kt [hunk 1]
```diff
@@ -538,14 +538,16 @@ class Notetypes(val col: Collection) {
             } else {
                 convertLegacyMap(cmap, newModel.templatesNames.size)
             }
+        val isCloze = newModel.isCloze || m.isCloze
```
Comment (david-allison) [inline AnkiDroid/src/main/java/com/ichi2/libanki/Notetypes.kt:541]: Tested :
* switching to cloze from basic & reversed
* switching back. 

General functionality seems good, I'm happy if tests are happy

**Review by BrayanDSO — APPROVED** [review]

**Comment by BrayanDSO** [comment]
Pending CI

**Comment by david-allison** [comment]
🥳🥳

**Pull request merged**